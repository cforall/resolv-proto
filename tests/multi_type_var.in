A baz A | B foo A | A bar B
#b foo #a
#a bar #b
%%
baz( #a )
