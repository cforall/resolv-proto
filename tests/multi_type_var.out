
[#a / (0,2,0)]{A<1> => #a, B<2> => #b | A<1> bar B<2> => bar( #b ) | B<2> foo A<1> => foo( #a )} baz{A<1>, B<2> | B<2> foo A<1> | A<1> bar B<2>}( #a )

[#a / (0,3,0)]{A<1> => #a, B<2> => #b | B<2> g A<1> => g( #a ) | A<1> h B<2> => h( #b )} f{A<1>, B<2> | B<2> g A<1> | A<1> h B<2>}( #a )
